package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha || true"
```

Comment:
The errors you are getting are caused by the errors that are in that same file(index.js). If I put the test function in its own file, I will not get that error.

<img width="1440" alt="Screen Shot 2019-04-24 at 20 09 25" src="https://user-images.githubusercontent.com/33568373/56683608-2d405e00-66ce-11e9-99b4-7272a61a1df8.png">

Also **|| true**  in the test script, I wrote it because I wanted to disable errors that were thrown when the test doesn't pass.
When I change the file. It will look like this

<img width="883" alt="Screen Shot 2019-04-24 at 20 01 56" src="https://user-images.githubusercontent.com/33568373/56683455-de92c400-66cd-11e9-9c72-38db8dcf0a1e.png">